yt_dlp/extractor/blackboardcollaborate.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import parse_iso8601
+
+
+class BlackboardCollaborateIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<zone>[a-z-]+)\.bbcollab\.com/(?:[a-z-]+/)+(?P<id>[^/]+)'
```

Comment:
Looks a bit broad, doesn’t it? The testcases only cover cases where the path before the video ID is `/collab/ui/session/playback/load/` and `/recording/`. Are there any others?

Reply:
I don't think there are others.